cmd/manager/main.go

```diff
@@ -2,53 +2,78 @@ package main
 
 import (
 	"context"
+	"errors"
 	"flag"
 	"fmt"
 	"os"
 	"runtime"
+	"strings"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"k8s.io/client-go/rest"
+
+	"github.com/percona/percona-server-mongodb-operator/pkg/apis"
+	"github.com/percona/percona-server-mongodb-operator/pkg/controller"
+	"github.com/percona/percona-server-mongodb-operator/version"
 
-	certmgrscheme "github.com/jetstack/cert-manager/pkg/client/clientset/versioned/scheme"
 	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	kubemetrics "github.com/operator-framework/operator-sdk/pkg/kube-metrics"
 	"github.com/operator-framework/operator-sdk/pkg/leader"
-	"github.com/operator-framework/operator-sdk/pkg/ready"
+	"github.com/operator-framework/operator-sdk/pkg/log/zap"
+	"github.com/operator-framework/operator-sdk/pkg/metrics"
 	sdkVersion "github.com/operator-framework/operator-sdk/version"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"github.com/spf13/pflag"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
-	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
-
-	"github.com/percona/percona-server-mongodb-operator/pkg/apis"
-	"github.com/percona/percona-server-mongodb-operator/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
 
+// Change below variables to serve metrics on different host or port.
 var (
-	GitCommit string
-	GitBranch string
-	log       = logf.Log.WithName("cmd")
+	metricsHost               = "0.0.0.0"
+	metricsPort         int32 = 8383
+	operatorMetricsPort int32 = 8686
```

Comment:
@qJkee is there any authorisation for accessing these ports?
@delgod are we ready to expose some HTTP ports in the operator? 

Anyway I suggesting consider it in the separate ticket/feature/PR